src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -43,39 +49,41 @@ public static String getAccessToken(User user) {
     return getAccessToken(user.getUserName(), user.getPassword());
   }
 
-  public static String getAccessToken(String userName, String password) {
-
-    String clientId = HeliosConfig.getClientId();
-    String clientSecret = HeliosConfig.getClientSecret();
-    String heliosGetTokenURL = HeliosConfig.getUrl(HeliosConfig.HeliosController.TOKEN);
-
-    RequestConfig requestConfig = RequestConfig.custom().setConnectTimeout(3000).setSocketTimeout(
-        3000).build();
+  public static void deleteAllTokens(User user) {
+    String heliosGetTokenURL = HeliosConfig.getUrl(HeliosConfig.HeliosController.USERS);
 
-    CloseableHttpClient httpClient =
-        HttpClientBuilder.create().disableCookieManagement().disableConnectionState()
-            .disableAutomaticRetries().build();
+    HttpDelete httpDelete =
+        new HttpDelete(String.format("%s/%s/tokens", heliosGetTokenURL, user.getUserId()));
+    httpDelete.setConfig(requestConfig);
+    httpDelete.setHeader("THE-SCHWARTZ", Configuration.getCredentials().apiToken);
 
+    CloseableHttpResponse response = null;
     try {
-      if (tokenCache.containsKey(userName)) {
+      response = getDefaultClient().execute(httpDelete);
 
-        String
-            getTokenInfoURL =
-            HeliosConfig.getUrl(HeliosConfig.HeliosController.INFO) + String
-                .format("?code=%s", tokenCache.get(userName));
-        HttpGet getInfo = new HttpGet(getTokenInfoURL);
-        getInfo.setConfig(requestConfig);
+      PageObjectLogging.log("DELETE HEADERS: ", response.toString(), true);
 
-        if (httpClient.execute(getInfo).getStatusLine().getStatusCode() == 200) {
-          return tokenCache.get(userName);
-        }
-      }
+    } catch (ClientProtocolException e) {
+      PageObjectLogging.log("CLIENT PROTOCOL EXCEPTION", ExceptionUtils.getStackTrace(e), false);
+      throw new WebDriverException(e);
     } catch (IOException e) {
       PageObjectLogging.log("IO EXCEPTION",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "IO EXCEPTION" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)